doc/release-notes/rl-2505.section.md

```diff
@@ -386,14 +450,70 @@
 
 - `gerbera` now has wavpack support.
 
+- `buildGoModule` now supports a self-referencing `finalAttrs:` parameter
+  containing the final arguments including overrides.
+  This allows packaging configuration to be overridden in a consistent manner by
+  providing an alternative to `rec {}` syntax.
+
+- Caddy can now be built with plugins by using `caddy.withPlugins`, a `passthru` function that accepts an attribute set as a parameter. The `plugins` argument represents a list of Caddy plugins, with each Caddy plugin being a versioned module. The `hash` argument represents the `vendorHash` of the resulting Caddy source code with the plugins added.
```

Comment:
Hm now this should be in NixOS, no?

Reply:
`caddy.withPlugins` is a Nixpkgs interface. I think the module excerpt here is just a way to illustrate it rather than say it's only for NixOS

Reply:
Ah, but then let's just remove the module stuff around it, it would be super confusing in the Nixpkgs context.